acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/TransactionSignerParamsSupplier.java

```diff
@@ -29,29 +29,41 @@
 
 public class TransactionSignerParamsSupplier {
 
-  private final HashicorpVault hashicorpVault;
+  private final HashicorpNode hashicorpNode;
   private final String azureKeyVault;
   private final Path multiKeySignerDirectory;
 
   public TransactionSignerParamsSupplier(
-      final HashicorpVault hashicorpVault,
+      final HashicorpNode hashicorpNode,
       final String azureKeyVault,
       final Path multiKeySignerDirectory) {
-    this.hashicorpVault = hashicorpVault;
+    this.hashicorpNode = hashicorpNode;
     this.azureKeyVault = azureKeyVault;
     this.multiKeySignerDirectory = multiKeySignerDirectory;
   }
 
   public Collection<String> get() {
     final ArrayList<String> params = new ArrayList<>();
-    if (hashicorpVault != null) {
+    if (hashicorpNode != null) {
       params.add("hashicorp-signer");
       params.add("--auth-file");
-      params.add(createVaultAuthFile(hashicorpVault.getVaultToken()).getAbsolutePath());
+      params.add(createVaultAuthFile(hashicorpNode.getVaultToken()).getAbsolutePath());
       params.add("--host");
-      params.add(hashicorpVault.getIpAddress());
+      params.add(hashicorpNode.getHost());
       params.add("--port");
-      params.add(String.valueOf(hashicorpVault.getPort()));
+      params.add(String.valueOf(hashicorpNode.getPort()));
+      final String tlsEnabledOption =
+          hashicorpNode.isTlsEnabled() ? "--tls-enabled" : "--no-tls-enabled";
```

Comment:
Since it is test dsl, I wanted to have both options available and tested. To connect to hashicorp, we have `tls-enabled` option as default because proper Hashicorp vault setup has TLS enabled by default.

Reply:
This bit is resolved, `--tls-enabled=false` to disable TLS to hashicorp. By default it is enabled.